reference/5.1/Microsoft.PowerShell.Core/About/about_Break.md

```diff
@@ -169,18 +155,75 @@ switch -regex ($var) {
 ```
 
 In this example, the `$var` variable is created and initialized to a string
-value of "word2". The Switch statement uses the **Regex** class to match the
+value of `word2`. The `switch` statement uses the **Regex** class to match the
 variable value first with the term "word2". (The **Regex** class is a regular
 expression Microsoft .NET Framework class.) Because the variable value and the
-first test in the `Switch` statement match, the first code block in the
-`Switch` statement runs.
+first test in the `switch` statement match, the first code block in the
+`switch` statement runs.
 
-When PowerShell reaches the first `Break` statement, the `Switch` statement
-exits. If the four `Break` statements are removed from the example, all four
-conditions are met. This example uses the `Break` statement to display results
+When PowerShell reaches the first `break` statement, the `switch` statement
+exits. If the four `break` statements are removed from the example, all four
+conditions are met. This example uses the `break` statement to display results
 when the most specific condition is met.
 
-## SEE ALSO
+## Using break in a trap statement
+
+If the final statement executed in the body of a trap statement is `break`,
+the error object is suppressed and the exception is re-thrown.
+
+The following example create a **DivideByZeroException** exception that is
+trapped using the `trap` statement.
+
+```powershell
+function test {
+  trap [DivideByZeroException] {
+    Write-Host 'divide by zero trapped'
+    break
+  }
+
+  $i = 3
+  'Before loop'
+  while ($true) {
+     "1 / $i = " + (1 / $i--)
+  }
+  'After loop'
+}
+test
+```
+
+Notice that execution stop at the exception. The `After loop` is never reached.
+The exception is re-thrown after the `trap` executes.
+
+```Output
+Before loop
+1 / 3 = 0.333333333333333
+1 / 2 = 0.5
+1 / 1 = 1
+divide by zero trapped
+Attempted to divide by zero.
+At line:10 char:6
++      "1 / $i = " + (1 / $i--)
++      ~~~~~~~~~~~~~~~~~~~~~~~~
+    + CategoryInfo          : NotSpecified: (:) [], ParentContainsErrorRecordException
+    + FullyQualifiedErrorId : RuntimeException
+```
+
+## Do not use break outside of a loop, switch or trap
+
+When `break` is used outside of a construct that directly supports it
+(loops, `switch`, `trap`), PowerShell looks _up the call stack_ for an
+enclosing construct. If it can't find an enclosing construct, the current
+runspace is quietly terminated.
+
+This means that functions and scripts that inadvertently use a `break` outside
+of an enclosing construct that supports it, can inadvertently terminate their
```

Comment:
The comma after "supports it" isn't necessary.

Reply:
Fixed